Vagrantfile

```diff
@@ -24,5 +24,13 @@ Vagrant.configure("2") do |config|
     #v.gui = true
   end
 
+  # override username to be evap instead of vagrant. Mounting with uid and gid is necessary as the provision script
```

Comment:
and some comments on the "why" would be nice.

Reply:
added something, please check again.